### ISSUE TITLE
关于历史帧 Key 放大机制（is_amplify_history）的问题请教

### ISSUE THREAD

**Issue by JingZhe-8** [PKU-YuanGroup/Helios#44]
各位 Helios 作者好，感谢开源和公开训练代码，受益很多。
我在复现过程中，对历史帧 Key 放大机制有一些疑问想请教。

我在 Helios 仓库（main 分支最新提交 fe7118d）里看到：

1、各阶段训练配置中（stage1/stage2/stage3），is_amplify_history 默认是 false。
2、但模型代码里实现了历史 K 放大逻辑（history_key_scale初始化是 torch.ones，max_scale=10）。
3、推理代码中，若加载 lora + partial 路径，会在加载 partial 前显式将 is_amplify_history 设为 true；而加载已 merge 的 transformer 路径时，会按照模型配置将 is_amplify_history 设为 flase。

我自己的复现现象：

1、当stage1训练开启 is_amplify_history=true 时，训练初期 loss 会飙升（首步约 1.x），首个验证视频可能出现马赛克/严重失真，随后训练会很快发散并崩溃。
2、当stage1训练保持 is_amplify_history=false 时，训练稳定，首个验证视频没有马赛克；但即使训练到等效数据量后，仍无法彻底解决 chunk 与 chunk 的衔接问题，视频里仍会有闪烁。

如果我对代码路径理解有偏差，也请您指正。
再次感谢你们的工作和开源支持！
